src/scripts/import/api.js

```diff
@@ -0,0 +1,34 @@
+var angular = require('angular')
+var FormData = require('form-data')
+
+module.exports = /* @ngInject */function ($http, ENDPOINT_URL) {
+  var baseImport = function (type, importData) {
+    return $http({
+      method: 'POST',
+      headers: {
+        'Content-Type': undefined
+      },
+      url: ENDPOINT_URL + '/import/' + type,
+      data: importData,
+      transformRequest: function (data) {
```

Comment:
You can define global transform, that is activated based on some config like `$http({formData: true})`